docs/units-and-global-variables.rst

```diff
@@ -328,3 +329,10 @@ for an interface type ``I``:
     interface identifier of the given interface ``I``. This identifier is defined as the ``XOR`` of all
     function selectors defined within the interface itself - excluding all inherited functions.
 
+The following properties are available for an integer type ``I``:
```

Comment:
Sorry for the late comment, I think `I` is a bit misleading as it is mostly used for `Interfaces` (just as above). Perhaps could have used `T` (as type) ?

Reply:
@axic I agree. There was another instance where I used `X` instead of `I`. I'll submit a new PR to fix both of them to `T`